lib/Smokeping/matchers/ExpLoss.pm

```diff
@@ -105,7 +105,10 @@ sub Test($$) {
     my $alfa = 1-0.01**(1/$hist);
 
     my $rising = $self->{param}{rising};
-    my $falling = (defined $self->{param}{falling} || $rising);
+    my $falling = $rising;
+    if(defined $self->{param}{falling}) {
+        $falling = $self->{param}{falling};
+    }
```

Comment:
I guess 

```suggestion
    my $falling = $self->{param}{falling} // $rising;
```

should be ok too then

Reply:
Sure, I think the idea is clear (using $param{falling} if it is defined and $rising otherwise); and there are plenty of syntactical ways to represent it. Your one-liner does exactly the same, so feel free to use it for the merge.